common/djangoapps/util/milestones_helpers.py

```diff
@@ -82,6 +82,15 @@ def remove_prerequisite_course(course_key, milestone):
     )
 
 
+def remove_course_references(course_key):
+    """
+    Remove course milestones and course content milestones that are referenced to specified course_key
+    """
+    if not is_prerequisite_courses_enabled():
```

Comment:
I suggest to do it in another manner:

```python
def remove_course_references(course_key):
    if is_prerequisite_courses_enabled():
        milestones_api.remove_course_references(course_key)
```

Reply:
Looks reasonable. 
It was taken from way this check is performed in milestone_helpers. Actually when I look at this file I think it would be good to refactor it - smth like take out checks to decorators instead of copy-paste to every function.